db/migrate/20210607151925_add_teacher_standards_to_lessons.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddTeacherStandardsToLessons < ActiveRecord::Migration[6.1]
+  def change
+    change_table :course_lessons, bulk: true do |t|
+      t.string :ect_teacher_standards, null: true
+      t.string :mentor_teacher_standards, null: true
```

Comment:
The bulk true is combining altar table queries, what's the benefit in doing that? or is this something that always happens and i've just not noticed it?

Reply:
Hehe, the benefit for me is shutting Rubucop up.

The slight benefit is that you don't need to specify the table name multiple times - `course_lessons` is just mentioned once.